src/britive/notification_mediums.py

```diff
@@ -0,0 +1,65 @@
+class NotificationMediums:
+    def __init__(self, britive):
+        self.britive = britive
+        self.base_url = f'{self.britive.base_url}/v1/notification-service/notificationmediums'
+    def list(self):
+        """
+        Provide a list of all notification mediums 
```

Comment:
List all notification mediums.

Reply:
with newline then :return

Reply:
`    def list(self) -> dict:
        """
        List all notification mediums.

        :return: List of all notification mediums
        """

        return self.britive.get(self.base_url)`

Fixed